databricks/koalas/frame.py

```diff
@@ -2099,6 +2099,7 @@ def apply_batch(self, func, args=(), **kwds):
         0  1  2
 
         You can also specify extra arguments.
+
```

Comment:
For some reasons, this style makes the doc build failed in my local:

```
Warning, treated as error:
/.../koalas/databricks/koalas/frame.py:docstring of databricks.koalas.DataFrame.apply_batch:109:Unexpected indentation.
make: *** [html] Error 2
```

:confused: